book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please   Getters and Setters for @rentals should be present, add rentals to attr_accessor.